token/common.tact

```diff
@@ -93,3 +93,33 @@ struct BurnNotification {
 
     @derive impl Serialize {}
 }
+
+contract interface JettonOwner {
```

Comment:
```suggestion
interface JettonOwner {
```

Reply:
Fixed in [1f63906](https://github.com/tact-lang/tact-bikeshed/pull/5/commits/1f63906469f35d23260a3ba0ab4480f450df9520)